sklearn/tests/test_dummy.py

```diff
@@ -617,3 +617,14 @@ def test_dummy_regressor_on_nan_value():
     clf.fit(X, y)
     y_pred = clf.predict(X)
     assert_array_equal(y_pred, y_expected)
+
+   
+def test_dummy_regressor_on_object_value():
+    X = np.array(['foo', 'bar', 'baz'])
```

Comment:
this won't actually have dtype object (check for yourself) so the test is incorrectly named.

Reply:
Sorry. I'll change it to string.